### ISSUE TITLE
The batch size for three training stages

### ISSUE THREAD

**Issue by hwei-hw** [jiangsongtao/TinyMed#7]
Could you provide the adopted batch size in three training stages? 
Below show the details of MoE-LLAVA in three stages. Are you adopt the same hyper-parameters? Thanks a lot!

> 
![image](https://github.com/jiangsongtao/TinyMed/assets/11232069/fd2d04ab-ea3b-4f3c-bcc3-edb201d39139)

**Comment by jiangsongtao** [comment]
Hello! Our training also consists of three stages. We use 8x3090 for training, the parameters are as follows: per_device_train_batch_size is set to 4, per_device_eval_batch_size is set to 2, and gradient_accumulation_steps is set to 2. The other parameters are consistent with the ones shown in the above figure.

**Comment by hwei-hw** [comment]
Thanks for your reply. And for the stage 1, I set these parameters as follows to train the stage 1 model at 2*A100:

>   --per_device_train_batch_size 16 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 8 \
    # other parameters as same as your project

Getting the loss curve:
![image](https://github.com/jiangsongtao/TinyMed/assets/11232069/421649e3-966b-4d62-b14e-eb5ed129830a)

The final value of the training loss is about 2.2. 

Based on your training, I wonder if this loss is normal?
Or, I should retrain it by your setting of `batch_size`?
Or, should I load the weights of Moe-llava during the training of stage 1?
And this setting `per_device_train_batch_size is set to 4, per_device_eval_batch_size is set to 2` suit for all three stages?

Thanks a lot and look forward your reply.

**Comment by hwei-hw** [comment]
By the way, I use this training script: `scripts/train_scripts/phi2/pretrain.sh`

**Comment by jiangsongtao** [comment]
Hello, during the pretraining stage of phi2, a loss around 2.2 is normal. Since your GPUs are 2*A100, I suggest using a larger batch size to better utilize the memory and improve training efficiency. Therefore, you don't need to strictly follow my training parameters. I have also uploaded my stage1 pretrain weight on https://huggingface.co/JsST/TinyMed/tree/main/Stage1, and you can load my stage1 weight as well. Note that since Moellava's stage1 data is not medical data, loading them might cause issues.

**Comment by hwei-hw** [comment]
Thanks a lot for your timely reply! I will try this.
